### PR TITLE
Cards tts json export

### PR DESCRIPTION
**Pull request by resonant-gamedesign**
Adds TTS export for cards

### CODE REVIEW

**Comment by resonant-gamedesign** [comment]
@neubee What do I do when the Linting fails? I don't have any errors on my side...

**Comment by tomprince** [comment]
> @neubee What do I do when the Linting fails? I don't have any errors on my side...

If you click the details beside the job (or the red `X` next to the commit), it will take you to the output of the job, where I see
```
[warn] src/routes/power-cards/tts-power-card.json
[warn] Code style issues found in the above file. Forgot to run Prettier?
```

Which says that the new json file needs formatting.

You can run prettier in the repo, by running `npm run writeFormatAndLint` or `npx prettier --write .`. In that second one, the `.` represents the current directory, and you can replace it by a subdirectory or file to just format that subdirectory/file.

**Comment by tomprince** [comment]
I wonder if it would make sense to export all the cards as a deck rather than individual cards? If we did that, it'd likely make sense to have an option to export the images as a single file in the format that TTS expects decks to be.

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
       "eslint",
       "prettier --write --plugin-search-dir=."
     ],
-    "*.{css}": [
+    "*.{css,json,html}": [
```
Comment (neubee) [inline package.json:41]: I just added this so `prettier` should format any json and html files on commit. Hopefully that reduces the number of prettier errors on the github end of things.

**Comment by resonant-gamedesign** [comment]
> I wonder if it would make sense to export all the cards as a deck rather than individual cards? If we did that, it'd likely make sense to have an option to export the images as a single file in the format that TTS expects decks to be.

You might have a better sense for how TTS works - and you are welcome to push to my branch (not sure if I have to set that up). I was frustrated with decks because I find it harder to change their images. I'm used to being able to right click and go to custom on TTS but my understanding is that you can't do that for decks. Thoughts?

File: package.json (hunk 1 above)
Comment (resonant-gamedesign) [reply package.json:41]: looks good!

**Pull request merged**